templates/sentinel-init.conf.erb

```diff
@@ -1,14 +1,90 @@
-# redis-sentinel - Redis Datastore Server
-#
-# Redis is a key value in memory persistent datastore
-
-start on (local-filesystems and runlevel [2345])
-stop on runlevel [016]
-respawn
-expect fork
-limit nofile 20000 65000
-pre-start script
-mkdir -p /var/run/redis-sentinel
-chown redis:redis /var/run/redis-sentinel
-end script
-exec start-stop-daemon --start --chuid redis:redis --pidfile <%= @conf_pidfile_real %> --umask 007 --exec /usr/bin/redis-sentinel -- <%= @conf_sentinel %>
+#! /bin/sh
```

Comment:
Can you make a version of the init script to work on RHEL systems as well ? So we can have sentinel for both RHEL/Debian platforms.

The start-stop-daemon is available only on Debian/Ubuntu. I think the best route is to create another init script for RHEL based systems and based in the OS assign the right template.